### PR TITLE
feat: setup firebase initial configuration in NestJS App

### PR DESCRIPTION
**Pull request by AlejandraRochaOMcKinsey**
## Issue ticket number and link

https://trello.com/c/yca7l7pX/330-inicio-configuraci%C3%B3n-firebase-con-nestjs
image

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [x] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

-  No integration with NestJS

## What is the new behavior?

- The current NestJS module provides  integration with Firebase

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```
[the remark this answers is not in the document]
Comment (FelipeLosada1996) [reply jest.config.js:3]: and the regex of what abbreviation is gonna test. 
The idea is that test all the files with `spec.ts`

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by Sofiag8 — APPROVED** [review]

**Review by kellyCSalazar — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "prepare": "husky install",
     "test:ci": "yarn test --coverage",
     "test:coverage": "yarn test --coverage",
+    "test-jest": "jest",
```
Comment (jose-sal-y-rosas) [inline package.json:15]: what is the difference between test:coverage and test-jest?

**Review by jose-sal-y-rosas — COMMENTED** [review]
Please review the comment

File: package.json (hunk 2 above)
Comment (AlejandraRochaOMcKinsey) [reply package.json:15]: In the project, "test:coverage" is configured to run Express tests, while "test-jest" is set up to run NestJS

Comment (jose-sal-y-rosas) [reply package.json:15]: Got it. Please add a comment with that description and change the command for something like test:unit:tests.

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**